services/src/main/java/org/exoplatform/task/rest/ProjectRestService.java

```diff
@@ -264,36 +265,67 @@ public Response getProjectsStatistics(@ApiParam(value = "id", required = true) @
   }
   }
 
-
   @GET
-  @Path("users/{query}/{projectName}")
+  @Path("users/{projectId}/{query}")
```

Comment:
This API change may require synchronizing with Task FT
Besides, I think you can use the function getProjectParticipants instead.

Reply:
Yes I checked before modifying and I didn't find any usage of it in the Task addon.
The `getProjectParticipants` rest return all participant and do not search with term. So I will modify and use it instead of `getUsersByQueryAndProjectName`.

Reply:
I also checked for the usage of `getProjectParticipants` and it is only used only on time with empty term.